components/ActiveLink/ActiveLink.js

```diff
@@ -0,0 +1,30 @@
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+export default function ActiveLink({ path, title }) {
+  const router = useRouter()
+  const className = path === router.pathname ? 'active' : ''
+
+  return (
+    <Link href={path} passHref>
+      <LinkStyled className={className}>{title}</LinkStyled>
+    </Link>
+  )
+}
+
+const LinkStyled = styled.a`
```

Comment:
Do we need to use an `anchor tag` here? I would expect that the `Link` from next is already an anchor tag. Maybe a span is sufficient then as this specific element does not actually link further? Or is this better considering accessibility?

Reply:
That's something I haven't figured out yet as well. In the next.js docs, they do it like this and I can't style the `Link` component as we know from react-router: `const LinkStyled = styled(Link)`.

Reply:
I added a card on Trello with issues I encountered: https://trello.com/c/9lnL8Bnz